[editor]/editor_main/server/freeroam.lua

```diff
@@ -6,8 +6,8 @@ addEventHandler("onResourceStart",getResourceRootElement(getThisResource()),
 		end
 
 		if getResourceState(freeroam) == "running" then
-			outputChatBox ( "WARNING: 'FREEROAM' resource is currently running.  The resource has been shut off as a precaution!", getRootElement(), 255, 0, 0 )
-			outputDebugString (  "WARNING: 'FREEROAM' resource is currently running.  The resource has been shut off as a precaution!" )
+			outputChatBox ( "INFO: 'FREEROAM' resource is currently running.  The resource has been shut off as a precaution!", getRootElement(), 255, 255, 0 )
+			outputDebugString (  "INFO: 'FREEROAM' resource is currently running.  The resource has been shut off as a precaution!" )

```

Comment:
```suggestion
			outputDebugString (  "'FREEROAM' resource is currently running.  The resource has been shut off as a precaution!" )

```
No need for 'info' here because it's prefixed automatically by outputDebugString